docs/reduction_toolbox.rst

```diff
@@ -428,6 +432,112 @@ which should be removed rather than propagated:
     plt.show()
 
 
+Working with multi-extension FITS image files
+---------------------------------------------
+
+Multi-extension FITS (MEF) image files cannot be processed natively in ``ccdproc``. The example below illustrates how to `~ccdproc.flat_correct` all of the extensions in a MEF and write out the calibrated file as a MEF. Applying other reduction steps would be similar.
```

Comment:
This is a very long line, could you add some newlines? I'm not sure about the complete doc but generally 80 or 100 chars per line is a good (but soft) maximum.